types/operator.go

```diff
@@ -62,14 +64,30 @@ func (o Operator) Validate() error {
 	return operatorMetadata.Validate()
 }
 
-// Should we also add the operator address in here?
+type Socket = string
```

Comment:
change this to type and remove aliases

Reply:
[8f1d3f1](https://github.com/Layr-Labs/eigensdk-go/pull/206/commits/8f1d3f1171eb2f97b4e749ab98c91e4db0650923)